program_v2/tests/sol-did.ts

```diff
@@ -47,15 +47,115 @@ describe("sol-did", () => {
     expect(rawDidDataAccount.data.length).to.equal(10_000)
 
 
-    console.log("Your transaction signature", tx);
+    console.log("Your transaction signature1", tx);
   });
 
+  //add data
+  it("Service added!", async () => {
```

Comment:
can you improve the test disciption e.g. `it("should successfully add a new service". Also review others. Thanks!